aws/logs_monitoring/lambda_function.py

```diff
@@ -374,15 +387,22 @@ def scrub(self, payload):
 
 def datadog_forwarder(event, context):
     """The actual lambda function entry point"""
+    global trace_connection
     events = parse(event, context)
-    metrics, logs = split(events)
+    metrics, logs, traces = split(events)
     if DD_FORWARD_LOG:
         forward_logs(filter_logs(logs))
     if DD_FORWARD_METRIC:
         forward_metrics(metrics)
+    if DD_FORWARD_TRACES and len(traces) > 0:
+        try:
+            for trace in traces:
+                trace_connection.send_trace(trace)
```

Comment:
Is `send_trace` blocking or non-blocking?

Reply:
Blocking